src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -179,13 +179,20 @@ function useChatUtilsHook(): UseChatUtilsHook {
     navigation.goBack();
   };
   const handleTextSystem = (item): string => {
+    let message = item?.description || item?.message;
+    if (message === ' ') {
+      message = 'sent media 🎆🌆🌉';
+    }
     if (
       item?.rawJson?.userIdFollower === profile?.myProfile?.user_id ||
       item?.rawJson?.message?.userIdFollower === profile?.myProfile?.user_id
     ) {
-      return item?.rawJson?.textOwnMessage || item?.rawJson?.message?.textOwnMessage;
+      message = item?.rawJson?.textOwnMessage || item?.rawJson?.message?.textOwnMessage;
+      if (message === ' ') {
+        message = 'sent media 🎆🌆🌉';
+      }
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider trimming the message before checking if it's empty.
```diff
-    if (message === ' ') {
+    if (message.trim() === '') {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->